traffic_ops/traffic_ops_golang/cdn/dnssec.go

```diff
@@ -245,7 +257,7 @@ type CDNDS struct {
 }
 
 // getCDNDeliveryServices returns basic data for the delivery services on the given CDN, as well as the CDN name, or any error.
-func getCDNDeliveryServices(tx *sql.Tx, cdn string) ([]CDNDS, string, error) {
+func GetCDNDeliveryServices(tx *sql.Tx, cdn string) ([]CDNDS, error) {
 	q := `
 SELECT ds.xml_id, ds.protocol, t.name as type, ds.routing_name, cdn.domain_name as cdn_domain
```

Comment:
The `cdn.domain_name as cdn_domain` should be removed from the query if it's not being scanned out from the result, right?

Reply:
Yep. Fixed.